doc/content/devices/models/milesight-em300-th/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Milesight EM300-TH"
+description: ""
+weight: 
+---
+
+{{< figure src="milesight-EM300-TH.png" alt="Milesight EM300-TH" class="float plain" width="75%" >}}
+
+The [Milesight EM300-TH](https://www.milesight.com/iot/product/lorawan-sensor/em300-th) is an environment sensor suitable for monitoring indoor and outdoor temperature and humidity. It can be widely used in office areas, greenhouses and other environments.
+
+<!--more-->
+
+## Provisioning Information
+
+The provisioning information, **DevEUI**, can be found on the back of the device. You can use `24E124C0002A0001` as the **JoinEUI**. The **AppKey** can be auto-generated within {{% tts %}}.
```

Comment:
```suggestion
The provisioning information can be found in various places:
The **DevEUI** is located on the back of the device. 
The **AppEUI/JoinEUI** is `24E124C0002A0001` by default.
The **AppKey** can be found inside of the email when purchasing device.
```

This is a rough suggestion, word it how you like, but the Appkey shouldnt be auto-generated, it is a value given with the device.

Reply:
@LDannijs Thanks for the info.